templates/firefox-ios.xliff

```diff
@@ -1362,6 +1366,30 @@
         <source>Filter</source>
         <note>Placeholder test for search box in logins list view.</note>
       </trans-unit>
+      <trans-unit id="LoginsList.NoLoginsFound.Description" xml:space="preserve">
+        <source>Saved logins will show up here. If you saved your logins to Firefox on a different device, sign in to your Firefox Account.</source>
+        <note>Label shown when there are no logins to list</note>
+      </trans-unit>
+      <trans-unit id="LoginsList.NoMatchingResult.Subtitle" xml:space="preserve">
+        <source>There are no results matching your search.</source>
+        <note>Label that appears after the search if there are no logins matching the search</note>
+      </trans-unit>
+      <trans-unit id="LoginsList.NoMatchingResult.Title" xml:space="preserve">
+        <source>No matching logins</source>
+        <note>Label displayed when a user searches and no matches can be found against the search query</note>
+      </trans-unit>
+      <trans-unit id="LoginsList.Search.Cancel" xml:space="preserve">
```

Comment:
Could you add a quick comment here as to what Cancel does?
I'm afraid some locales would translate that word differently depending on context